### PR TITLE
[Term Entry] PHP Math Function: atan

### PR DESCRIPTION
**Pull request by hitesh-gitdev**
### Description

- Added the atan() term for PHP
- Closes issue #2205 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+Title: 'atan()'
+Description: 'Returns the angle whose tangent value is equal to the specified value.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Math Functions'
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:8]:
```suggestion
```
Not a subject in [the subjects.md file](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Tags:
+  - 'Methods'
+  - 'Trigonometric'
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:11]:
```suggestion
  - 'Trigonometry'
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The inverse tangent function, which is used to identify the angle whose tangent value is equal to the specified value, is referred to as the **arc tangent** function.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:18]:
```suggestion
The inverse tangent function, which is used to identify the angle whose tangent value is equal to the specified value, is referred to as the arc tangent function.
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+The built-in math function **`atan()`** determines the arc tangent of a given value. It is the inverse function of the [**`tan()`**](https://www.codecademy.com/resources/docs/php/math-functions/tan) function.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:20]:
```suggestion
The built-in math function **`atan()`** determines the arc tangent of a given value. It is the inverse function of the [`tan()`](https://www.codecademy.com/resources/docs/php/math-functions/tan) function.
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  - 'Methods'
+  - 'Trigonometric'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The inverse tangent function, which is used to identify the angle whose tangent value is equal to the specified value, is referred to as the **arc tangent** function.
+
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:19]:
```suggestion
```
Remove line break so these are the same paragraph.

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+The built-in math function **`atan()`** determines the arc tangent of a given value. It is the inverse function of the [**`tan()`**](https://www.codecademy.com/resources/docs/php/math-functions/tan) function.
+
+> This means that `tan(atan(number))` would be equal to `number`.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:22]:
```suggestion
This means that `tan(atan(number))` would be equal to `number`.
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+The syntax of the `atan()` function is as follows:
+
+```pseudo
+$result = atan(number);
+```
+
+The atan() function takes one parameter:
+
+- $number (required): The number for which you want to calculate the arc tangent.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:34]:
```suggestion
- $number (required): The number for which the arc tangent is to be calculated.
```
We avoid first- and second-person pronouns in docs.

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+Where `number` is the number whose arc tangent the function returns.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:36]:
```suggestion
```
This is redundant.

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+## Return Value
+
+The `atan()` method returns a floating point number which is the arc tangent value of the number passed as argument.
+
+The resulting numeric value is between **-pi/2** and **pi/2** radians.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:42]:
```suggestion
The `atan()` method returns a floating point number which is the arc tangent value of the number passed as argument. The resulting numeric value is between -pi/2 and pi/2 radians.
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+To convert radians to degrees, we can use the following:
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:44]:
```suggestion
To convert radians to degrees, use the following:
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+```
+degrees = radians * ( 180.0 / M_PI )
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:47]:
```suggestion
$degrees = $radians * ( 180.0 / M_PI )
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+
+This will result in the following output:
+
+```shell
+atan(0.5): 0.46364760900081 Radians
+```
+
+## Codebyte Example
+
+In the below example, let's use the `atan()` function to obtain the arc tangent value of a number in radians, and then convert it into degrees.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:66]:
```suggestion
The below example uses the `atan()` function to obtain the arc tangent value of a number in radians, and then converts it into degrees.
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+
+To convert radians to degrees, we can use the following:
+
+```
+degrees = radians * ( 180.0 / M_PI )
+```
+
+## Example
+
+```php
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/atan/atan.md:52]: Even with a simple example we like an introductory sentence summarizing what it is doing.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@hitesh-gitdev I did a first review and left a number of comments.

File: content/php/concepts/math-functions/terms/atan/atan.md (hunk 13 above)
Comment (hitesh-gitdev) [reply content/php/concepts/math-functions/terms/atan/atan.md:45]: Added an introductory sentence.

**Comment by hitesh-gitdev** [comment]
@SSwiniarski Addressed PR comments. Requesting another review.

**Review by SSwiniarski — APPROVED** [review]
@hitesh-gitdev Looks good for a second review.

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
+---
+Title: 'atan()'
+Description: 'Returns the angle whose tangent value is equal to the specified value.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+  - 'Computer Science'
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:7]:
```suggestion
```

Comment (caupolicandiaz) [reply content/php/concepts/math-functions/terms/atan/atan.md:7]: we only list 2 items under subjects

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
+---
+Title: 'atan()'
+Description: 'Returns the angle whose tangent value is equal to the specified value.'
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:3]:
```suggestion
Description: 'Returns the angle whose tangent is equal to the specified value.'
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+Tags:
+  - 'Methods'
+  - 'Trigonometry'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The inverse tangent function, which is used to identify the angle whose tangent value is equal to the specified value, is referred to as the arc tangent function. The built-in math function **`atan()`** determines the arc tangent of a given value. It is the inverse function of the [`tan()`](https://www.codecademy.com/resources/docs/php/math-functions/tan) function.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:17]:
```suggestion
The built-in math function **`atan()`** determines the arc tangent of a given value. It is also known as the inverse tangent function, it identifies the angle whose tangent is equal to the specified value. The inverse function of `atan() is the [`tan()`](https://www.codecademy.com/resources/docs/php/math-functions/tan) function.
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+
+This means that `tan(atan(number))` would be equal to `number`.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:19]:
```suggestion
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+
+## Syntax
+
+The syntax of the `atan()` function is as follows:
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:23]:
```suggestion
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+
+```pseudo
+$result = atan(number);
+```
+
+The atan() function takes one parameter:
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:29]:
```suggestion
The `atan()` function takes one parameter:
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+The inverse tangent function, which is used to identify the angle whose tangent value is equal to the specified value, is referred to as the arc tangent function. The built-in math function **`atan()`** determines the arc tangent of a given value. It is the inverse function of the [`tan()`](https://www.codecademy.com/resources/docs/php/math-functions/tan) function.
+
+This means that `tan(atan(number))` would be equal to `number`.
+
+## Syntax
+
+The syntax of the `atan()` function is as follows:
+
+```pseudo
+$result = atan(number);
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:26]:
```suggestion
$result = atan($number);
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+The atan() function takes one parameter:
+
+- $number (required): The number for which the arc tangent is to be calculated.
+
+The `atan()` method returns a floating point number which is the arc tangent value of the number passed as argument. The resulting numeric value is between **-pi/2** and **pi/2** radians.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:33]:
```suggestion
The `atan()` method returns a floating point number which is the arc tangent value of the number passed. The resulting numeric value will be between **-pi/2** and **pi/2** radians.
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+
+To convert radians to degrees, use the following:
+
+```
+$degrees = $radians * ( 180.0 / M_PI )
+```
+
+## Example
+
+The below example illustrates the use of the `atan()` function to calculate the arc tangent of a number and print the result.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:43]:
```suggestion
The example below illustrates the use of the `atan()` function to calculate the arc tangent of a number and print the result.
```

File: content/php/concepts/math-functions/terms/atan/atan.md [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+
+This will result in the following output:
+
+```shell
+atan(0.5): 0.46364760900081 Radians
+```
+
+## Codebyte Example
+
+The below example uses the `atan()` function to obtain the arc tangent value of a number in radians, and then converts it into degrees.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan/atan.md:59]:
```suggestion
The example below uses the `atan()` function to obtain the arc tangent value of a number in radians, and then converts it into degrees.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@hitesh-gitdev I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz Addressed comments from second review. Requesting another review.
> I've rebased, squashed the PR edit commits, and pushed changes. I hope that's okay?

In the future I would recommend leaving rebasing, merging, etc to maintainers. It's also easier to review changes if the history hasn't been collapsed.

**Review by caupolicandiaz — APPROVED** [review]
@hitesh-gitdev I've made some additional minor edits. Looks good to go.

**Comment by SSwiniarski** [comment]
@hitesh-gitdev Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/math-functions/atan

image

**Comment by hitesh-gitdev** [comment]
Woohoo! Thank you so much @SSwiniarski and @caupolicandiaz. In a way, also my first contribution to an Open Source project in my entire SWE career. 😂

**Pull request merged**